backend/server.js

```diff
@@ -867,7 +639,7 @@ async function processData(data, query, callback) {
 
         /* else {
```

Comment:
Can we delete the commented code altogether?

Reply:
Yup.